en/docs/learn/tutorials/expose-a-soap-service-as-a-rest-api.md

```diff
@@ -59,6 +59,16 @@ This tutorial will explain the steps to design, publish and invoke a SOAP servic
      see [Enforce Throttling and Resource Access Policies]({{base_path}}/learn/rate-limiting/setting-throttling-limits/).</p>
      </div>
      </html>
+
+    <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
```

Comment:
I think as @kavishkafernando has mentioned in the [issue](https://github.com/wso2/docs-apim/issues/1584), It's better to add a new step before this
```
6. Navigate to the Resources tab to view and edit you in/out sequences of the API.
```
![image](https://user-images.githubusercontent.com/3313885/102918006-32117280-44ac-11eb-97d9-6e9a7505f807.png)
we can move current 6th step to step 7

And also shall we add a screenshot as well ?
The screenshot should show the sequence editor on the resources page.

Reply:
I think 3.1.0 does not have the 6th step at all , Above screenshot is from 3.2.0